_includes/head.html

```diff
@@ -1,22 +1,65 @@
-<!-- meta -->
+
 <meta charset={{ site.encoding }}>
 <meta http-equiv="X-UA-Compatible" content="IE=edge">
 <meta name="viewport" content="width=device-width, initial-scale=1.0, shrink-to-fit=no">
+
+<title>{{ page.title }}</title>
 <meta name="description" content="{{ site.description }}">
 <meta name="author" content="{{ site.author }}">
-<title>
-  {% if page.type == "podcast" %}
-    {% assign title_length = page.title | size %}
-    {{ page.title | slice: 7, title_length }} - The Safing Podcast #{{ page.title | slice: 2, 2 }}
-  {% elsif page.type == "video" %}
-    {% assign title_length = page.title | size %}
-    {{ page.title | slice: 7, title_length }} - Safing Update #{{ page.title | slice: 2, 2 }}
-  {% elsif page.type == "post" %}
-    {{ page.title }} - Safing Blog
+
+<!-- Other & Facebook -->
+<meta property="og:url" content="{{ site.url }}">
+<meta property="og:type" content="website">
+<meta property="og:title" content="{{ page.title }}">
+<meta property="og:description" content="
+  {% if page.summary != null %}
+    {{ page.summary | truncatewords: 18 }}
+  {% else %}
+    {{ site.description }}
+  {% endif %}
+">
+<meta property="og:image" content="
+  {% if page.category == "podcast" %}
+    {% if page.custom_thumbnail_name == null %}
+      {{ site.url }}{{ site.img_url }}thumbnails/podcast/placeholder.png
```

Comment:
Is the file correct? The filename in the PR is `placeholder_S02.png`.

Reply:
Yes